Adyen/Service/Checkout/ClassicCheckoutSDKService.cs

```diff
@@ -22,75 +22,89 @@
 
 namespace Adyen.Service.Checkout
 {
+    /// <summary>
+    /// ClassicCheckoutSDKService Interface
+    /// </summary>
+    public interface IClassicCheckoutSDKService
+    {
+        /// <summary>
+        /// Create a payment session
+        /// </summary>
+        /// <param name="paymentSetupRequest"><see cref="PaymentSetupRequest"/> - </param>
```

Comment:
@jillingk `paymentRequest -` has no description (while e.g. requestOptoins does have a descriptions), checking whether this was intended. Final suggestion to just add leave it as a `><see cref="PaymentSetupRequest"/>.` since the class PaymentSetupRequest will have the description